docs/admin/bootstrap-tokens.md

```diff
@@ -0,0 +1,167 @@
+---
+assignees:
+- jbeda
+title: Authenticating with Bootstrap Tokens
+---
+
+* TOC
+{:toc}
+
+## Overview
+
+Bootstrap tokens are a simple bearer token that is meant to be used when
+creating new clusters or joining new nodes to an existing cluster.  It was built
+to support [`kubeadm`](/docs/admin/kubeadm/), but can be used in other contexts.
+It is also built to work, via RBAC policy, with the [Kubelet TLS
+Bootstrap](/docs/admin/kubelet-tls-bootstrap/) system.
+
+Bootstrap Tokens are defined with a specific type
+(`bootstrap.kubernetes.io/token`) of secrets that lives in the `kube-system`
+namespace.  These Secrets are then read by the Bootstrap Authenticator in the
+API Server.  Expired tokens are removed with the TokenCleaner controller in the
+Controller Manager.  The tokens are also used to create a signature for a
+specific ConfigMap used in a "discovery" process through a BootstrapSigner
+controller.
+
+Currently, Bootstrap Tokens are **alpha** but there are no large breaking
+changes expected.
+
+## Token Format
+
+Bootstrap Tokens take the form of `abcdef.0123456789abcdef`.  More formally,
+they must match the regular expression `[a-z0-9]{6}\.[a-z0-9]{16}`.
+
+The first part of the token is the "Token ID" and is considered public
+information.  It is used when referring to a token without leaking the secret
+part used for authentication. The second part is the "Token Secret" and should
+only be shared with trusted parties.
+
+## Enabling Bootstrap Tokens
+
+All features for Bootstrap Tokens are disabled by default in Kubernetes v1.6.
+
+You can enable the Bootstrap Token authenticator with the
+`--experimental-bootstrap-token-auth` flag on the API server.  You can enable
```

Comment:
@lukemarsden Note that this doc isn't the kubeadm documentation, but the actual Bootstrap Token documentation. Hence this isn't directly coupled to kubeadm -- kubeadm is just the first consumer of this flow